lib/spack/spack/analyzers/environmentvariables.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+"""An environment analyzer will read and parse the environment variables
+file in the installed package directory, generating a json file that has
+an index of key, value pairs for environment variables."""
+
+
+import spack.monitor
+from .analyzerbase import Analyzerbase
+
+import os
+import re
+
+
+class Environmentvariables(Analyzerbase):
```

Comment:
any reason not to call this `environment_variables.py` and the class `EnvironmentVariables`?

Reply:
I like that better.

Reply:
I'm going to do it for the others too.